test/runner.zig

```diff
@@ -279,7 +279,7 @@ pub fn main() !void {
                 const expected_error = buf.items;
 
                 const index = std.mem.indexOf(u8, m.buf.items, expected_error);
-                if (index == null or m.buf.items[index.? + expected_error.len] != '\n') {
+                if (index == null) {
```

Comment:
I am not too sure about this modifications here so please let me know. But basically the two options are: 1) Change all existing EXPECT_ERRORS (or) 2) Don't check till eof, but I am not too sure for what reasons this limitation exists.